bundle/manifests/networking.istio.io_destinationrules.yaml

```diff
@@ -4854,9 +4816,6 @@ spec:
                           idleTimeout:
                             description: The idle timeout for TCP connections.
                             type: string
-                            x-kubernetes-validations:
-                            - message: must be a valid duration greater than 1ms
```

Comment:
do we know why this validation disappears?

Reply:
It was removed here - https://github.com/istio/istio/commit/5faf138b5a3a86d5ada52869a7d3e7353c904f25 checking.

Reply:
Here - https://github.com/istio/api/pull/3455